examples/rancher-gen/nginx.tmpl

```diff
@@ -0,0 +1,183 @@
+{{$local := host}}
+#This Host:
+#  UUID: {{$local.UUID}}
+#  Name: {{$local.Name}}
+#  Address: {{$local.Address}}
+#  Hostname: {{$local.Hostname}}
+#  Labels: {{$local.Labels}}
+
+# Access Log Formatting
+log_format vhost '$host $remote_addr - $remote_user [$time_local] '
+                 '"$request" $status $body_bytes_sent '
+                 '"$http_referer" "$http_user_agent"';
+
+
+# If we receive X-Forwarded-Proto, pass it through; otherwise, pass along the
+# scheme used to connect to this server
+map $http_x_forwarded_proto $proxy_x_forwarded_proto {
```

Comment:
1. Don't need Host block (lines 1-7), this was for debug/ID
1. Access log formatting needed for tracking traffic source/dest through the Nginx container
1. Not sure about X-Forwarded-Proto, taken from [docker-gen](https://github.com/jwilder/docker-gen)